modules/core/src/main/scala/org/tessellation/l0/http/Routes.scala

```diff
@@ -0,0 +1,29 @@
+package org.tessellation.l0.http
+import cats.effect.Async
+import cats.effect.std.Queue
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import org.tessellation.dag.domain.block.DAGBlock
+import org.tessellation.l0.domain.snapshot.SnapshotService
+import org.tessellation.security.signature.Signed
+
+import org.http4s.HttpRoutes
+import org.http4s.circe.CirceEntityCodec.circeEntityDecoder
+import org.http4s.dsl.Http4sDsl
+
+final case class Routes[F[_]: Async](
```

Comment:
please name it `SnapshotRoutes` or sth (same as the others that @TheMMaciek moved to sdk)